cron_jobs/cronUniversityPayments.ts

```diff
@@ -9,7 +9,7 @@ import dbUniversities from '../db/universities';
 dotenv.config();
 
 const sendMailToUniversities = async () => {
-  const allUniversities = await dbUniversities.getUniversities();
+  const allUniversities = await dbUniversities.getAllOrderByName();
```

Comment:
With this rename, we see that we always use ordered university list even if the order doesn't matter. We may add a `getAll` without order and use it in these cases.

Reply:
https://trello.com/c/awpSIF1Q/186-getalluniversities-always-order-universities